README.md

```diff
@@ -27,7 +27,7 @@ The Utility Connect widget can be instantiated for one of two user flows:
 
 The user flow is selected with the `config.scope` parameter described in the component API Reference below.
 
-Note that instantiating the Utility Connect widget requires a one-time-use OAuth access token in order to authenticate API requests. See [Creating OAuth Tokens](https://developers.arcadia.com/#operation/createOAuthToken) for instructions. Note that the type of token to use (`grant_type`) differs if you are creating a user or updating a user -- see `config.accessToken` in the component API Reference below for details.
+Note that instantiating the Utility Connect widget requires a one-time-use OAuth access token in order to authenticate API requests. See [Creating OAuth Tokens](https://developers.arcadia.com/#operation/createApiAccessToken) for instructions. Note that the type of token to use (`grant_type`) differs if you are creating a user or updating a user -- see `config.accessToken` in the component API Reference below for details.
```

Comment:
I think we're trying to avoid mentions of `OAuth`. Should we change this from `one-time-use Oauth access token` to `one-time-use authentication token`?

Reply:
Same comment about the link in the subsequent sentence, `Creating Oauth Tokens`. Perhaps we should say `Creating Authentication Tokens`?

Reply:
In the third sentence, I believe we took out `grant_type`. @dlopuch can you provide clarity here?

Reply:
This is more so related to what I just put up: https://github.com/ArcadiaPower/arcadia-power/pull/12315 - but yeah I think we can just remove OAuth and refer to this as the `utility_connect_token`

Reply:
The API documentation for the endpoint here will be added in [EN-330](https://arcadiapower.atlassian.net/browse/EN-330) (currently in process), so may make sense to wait until that is up to finalize... but for now:

```suggestion
Note that instantiating the Utility Connect widget requires a Utility Connect token in order to authenticate API requests. See [Creating Utility Connect Tokens](TKTK) for instructions. Note that an Access Token will not work for this purpose.
```

Reply:
Updated it to clif's version